vaadin-date-time-picker-flow/pom.xml

```
@@ -116,7 +116,7 @@
         <dependency>
             <groupId>org.webjars.bowergithub.vaadin</groupId>
             <artifactId>vaadin-text-field</artifactId>
-            <version>2.4.14</version>
+            <version>2.6.0-alpha4</version>
```

Update to stable version.
```suggestion
            <version>2.6.0</version>
```